src/features/account/components/reviews/ReviewsTabPanel.tsx

```diff
@@ -0,0 +1,12 @@
+import { SimpleGrid } from '@chakra-ui/react';
+import type { FunctionComponent } from 'react';
+
+export const ReviewsTabPanel: FunctionComponent = () => {
```

Comment:
How is this used?

Reply:
it will have more components in the up coming story, for demo purpose , we leave it empty